specification/vmwarecloudsimple/resource-manager/Microsoft.VMwareCloudSimple/stable/2019-04-01/vmwarecloudsimple.json

```diff
@@ -0,0 +1,3604 @@
+          },
+          "default": {
+            "description": "BadRequest",
+            "schema": {
+              "$ref": "#/definitions/CSRPError"
+            },
+            "headers": {
+              "Content-Type": {
+                "type": "string",
+                "default": "application/json"
+              }
+            }
+          }
+        },
+        "x-ms-examples": {
+          "StopInBodyVirtualMachine": {
+            "$ref": "./examples/StopInBodyVirtualMachine.json"
+          },
+          "StopInQueryVirtualMachine": {
+            "$ref": "./examples/StopInQueryVirtualMachine.json"
+          }
+        },
+        "x-ms-long-running-operation": true
+      }
+    }
+  },
+  "definitions": {
+    "AvailableOperation": {
+      "description": "Resource provider available operation model",
+      "properties": {
+        "display": {
+          "description": "The list of operations",
+          "$ref": "#/definitions/AvailableOperationDisplay"
+        },
+        "isDataAction": {
+          "description": "Indicating whether the operation is a data action or not",
+          "type": "boolean",
+          "default": false
+        },
+        "name": {
+          "description": "{resourceProviderNamespace}/{resourceType}/{read|write|delete|action}",
+          "type": "string"
+        },
+        "origin": {
+          "description": "The origin of operation",
+          "type": "string",
+          "enum": [
+            "user",
+            "system",
+            "user,system"
+          ],
+          "x-ms-enum": {
+            "modelAsString": false,
+            "name": "OperationOrigin"
+          }
+        },
+        "properties": {
+          "description": "The list of operation properties",
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/AvailableOperationDisplayPropertyServiceSpecification"
+        }
+      }
+    },
+    "AvailableOperationDisplay": {
+      "description": "Resource provider available operation display model",
+      "type": "object",
+      "properties": {
+        "description": {
+          "description": "Description of the operation for display purposes",
+          "type": "string"
+        },
+        "operation": {
+          "description": "Name of the operation for display purposes",
+          "type": "string"
+        },
+        "provider": {
+          "description": "Name of the provider for display purposes",
+          "type": "string"
+        },
+        "resource": {
+          "description": "Name of the resource type for display purposes",
+          "type": "string"
+        }
+      }
+    },
+    "AvailableOperationDisplayPropertyServiceSpecification": {
+      "description": "Available operation display property service specification model",
+      "properties": {
+        "serviceSpecification": {
+          "description": "The list of specification's service metrics",
+          "$ref":
```

Comment:
Move API need not be described in RP swagger. Looks like the request body wasnt removed.